contrib/examples/prometheus/prometheus.yml

```diff
@@ -63,7 +63,7 @@ spec:
         - "--storage.tsdb.retention=24h"
         ports:
         - containerPort: 9090
-          protocol: TCP
+          hostPort: 9090
```

Comment:
This is just for my own curiosity, but why change from protocol to hostport? It's not immediately obvious to me what that accomplishes.

Reply:
sloppy on my part - temporary for debug/easy of use, I added (well actually replaced protocol which defaults to TCP anyway) hostPort which causes the container to expose a port on your host so you can access the pod's endpoint from the native side (outside of the cluster).  I've reverted this change, thanks for the keen set of eyes!